src/github.com/getlantern/yamlconf/yamlconf.go

```diff
@@ -103,6 +103,14 @@ type Manager struct {
 	// example for fetching config updates from a remote server.
 	CustomPoll func(currentCfg Config) (mutate func(cfg Config) error, waitTime time.Duration, err error)
 
+	// Obfuscate: if true, the on-disk version of the config will be obfuscated by
+	// encrypting it using AES-128 encryption.
```

Comment:
Oh, here's the benchmark - https://gist.github.com/oxtoacart/7e9b43efd08f6625a188bf48f8a2e0f3